genomic_data_service/region_indexer_task.py

```diff
@@ -317,6 +317,20 @@ def list_targets(dataset):
     return target_labels
 
 
+def get_target_label(dataset):
+    target_label = None
+
+    target = dataset.get('target', dataset.get('targets'))
+    if target:
+        if isinstance(target, dict):
+            target_label = target['label']
+
+        if isinstance(target, list) and target:
+            target_label = target[0]['label']
```

Comment:
I am not sure this is correct, shouldn't it return a list of target_labels?  IIRC, targets is an array for certain Annotation objects.